test/run_pass/test_ble_setservices_central.js

```diff
@@ -0,0 +1,132 @@
+/* Copyright 2016-present Samsung Electronics Co., Ltd. and other contributors
```

Comment:
The first number is for when the file is created not when the project started. FYI iot.js itself started 2015.